docs/source/testing/mocking.md

```diff
@@ -219,57 +230,45 @@ const mocks = {
 };
 
 const server = new ApolloServer({
-  typeDefs,
-  resolvers,
-  mocks,
-  mockEntireSchema: false, // highlight-line
+  schema: addMocksToSchema({
+    schema: makeExecutableSchema({ typeDefs, resolvers }),
+    mocks,
+    preserveResolvers: true, // highlight-line
+  }),
 });
 
-server.listen().then(({ url }) => {
-  console.log(`🚀 Server ready at ${url}`)
-});
+const { url } = await startStandaloneServer(server, { listen: { port: 4000 } });
+
+console.log(`🚀 Server listening at: ${url}`);
 ```
 
-Running the above example with the `mockEntireSchema` option set to `false`, the `resolved` field now returns the string `Resolved`.
+Above, the `resolved` query now uses its defined resolver, so it returns the string `Resolved`.
 
 ## Mocking a schema using introspection
```

Comment:
I don't really know what the point of this section is. You want to run a mock server that mocks something you get by introspecting a different server? When would you want that? I'd delete it.

Reply:
Done!